src/gsys_pipeline.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+
+from log_parser import EventRule
+from event_parser import EventGroupRule
+from signals import *
+from evaluators import *
+from criteria import Criteria
+from sifter_builder import SifterBuilder
+from gsyslyzer_cli import Gsyslyzer
+
+sifter_builder = SifterBuilder()
```

Comment:
Putting all this code at the module level isn't the best. 

If this script is only intended to be executed directly, then all of this should probably go into the __main__ block. Or it should all go into a function that the __main__ block calls.